docs/conf.py

```diff
@@ -1,159 +1,31 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-#
-# Python-Apollo documentation build configuration file, created by
-# sphinx-quickstart on Tue May  9 19:06:39 2017.
-#
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+import os
+import sys
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+from recommonmark.parser import CommonMarkParser
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'apollo'))
 
-# -- General configuration ------------------------------------------------
-
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
-
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = ['sphinx.ext.autodoc',
-    'sphinx.ext.coverage',
-    'sphinx.ext.viewcode']
-
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
-
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
-
-# The master toctree document.
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.napoleon']
 master_doc = 'index'
 
-# General information about the project.
-project = 'Python-Apollo'
```

Comment:
is it fine without project title?

Reply:
I think it's not needed, readthedocs must be guessing it automatically.
I mostly copied the conf.py from python-chado and python-tripal

Reply:
sounds good